src/main/java/com/example/coursework2/service/JavaQuestionService.java

```diff
@@ -0,0 +1,60 @@
+package com.example.coursework2.service;
+
+import com.example.coursework2.Question;
+import com.example.coursework2.exeption.QuestionNumberException;
+import com.example.coursework2.interfaces.QuestionService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+@Service
+public class JavaQuestionService implements QuestionService {
+
+    private static final List<Question> questionList = new ArrayList<>();
+    private static final Random random = new Random();
+
+    @Override
+    public Question questionAdd(String question, String answer) {
+        Question newQuestion = new Question(question, answer);
+        if (questionList.contains(newQuestion)){
+            throw new QuestionNumberException("Такой вопрос уже есть");
+        }
+        questionList.add(newQuestion);
+        return newQuestion;
```

Comment:
дублирование кода, надо было просто сделать делегацию
```suggestion
        return questionAdd(new Question(question, answer));
```